### PR TITLE
Improved graphics

### PR DESCRIPTION
**Pull request by lukipuki**
- solid curves
- dashed line only goes to the maximum and has the same color as the curve
- green zone is less opaque
- the blue background is gone
- vertical grid lines are gone

Screenshot_20200417_230634

### CODE REVIEW

File: covid_graphs/covid_graphs/country_graph.py [hunk 1]
```diff
@@ -23,10 +23,10 @@ def __str__(self):
...
     """Constructs a graph for a given country"""
     def __init__(
-        self,
-        data_dir: Path,
-        country_predictions: List[CountryPrediction],
-        graph_type: GraphType = GraphType.Normal,
+            self,
+            data_dir: Path,
+            country_predictions: List[CountryPrediction],
+            graph_type: GraphType = GraphType.Normal,
```
Comment (miskosz) [inline covid_graphs/covid_graphs/country_graph.py:29]: Something fishy happened here :)

File: covid_graphs/covid_graphs/country_graph.py [hunk 2]
```diff
@@ -83,28 +85,35 @@ def create_country_figure(self):
...
+        layout = Layout(
+            title=f"Active cases in {self.name}",
+            xaxis=dict(
+                autorange=True,
+                title=f'Day [starting at the {self.min_case_count}th case]',
+                showgrid=False,
+            ),
+            yaxis=dict(
+                autorange=True,
+                tickformat='.0f',
```
Comment (miskosz) [inline covid_graphs/covid_graphs/country_graph.py]: Interesting, does this print e.g. "80k" as in your screenshot? From the docs I thought it should be `"~s"`: https://github.com/d3/d3-format/blob/master/README.md#locale_format
(Also I like the "80k" format.)

File: covid_graphs/covid_graphs/scripts/country_dashboard.py [hunk 3]
```diff
@@ -66,9 +66,8 @@ def _get_header_content(prediction_event: PredictionEvent, title: str):
             'On 2020-03-30, mathematicians Katarína Boďová and Richard Kollár ',
             html.A('made predictions about 7 countries', href=prediction_link),
             f'. The data available up to that point (until {prediction_date_str}) is in the ',
-            html.Span('green zone', style={'color': 'green'}),
-            f'. Data coming after {prediction_date_str} is in the ',
-            html.Span('blue zone.', style=dict(color='blue')),
+            html.Span('light green zone', style={'color': 'lightgreen'}),
```
Comment (miskosz) [inline covid_graphs/covid_graphs/scripts/country_dashboard.py]: Just wondering if `background-color` attribute would be more fitting.

**Review by miskosz — APPROVED** [review]
❤️

File: covid_graphs/covid_graphs/country_graph.py [hunk 4]
```diff
@@ -62,15 +62,17 @@ def __init__(
...
 
     def create_country_figure(self):
+        colors = ['SteelBlue', 'Purple', 'Green'][:len(self.curves)]
+
         shapes = [
             # Add vertical dotted lines marking the maxima
             dict(type="line",
-                 yref="paper",
                  x0=curve.t[curve.maximal_idx],
                  y0=0,
```
Comment (miskosz) [inline covid_graphs/covid_graphs/country_graph.py:71]: Nemali by sme dat y0>0? (Napr. `y0=1`.) Mam pocit ze semilog a loglog grafy vyzeraju divne...

File: covid_graphs/covid_graphs/country_graph.py (hunk 1 above)
Comment (lukipuki) [reply covid_graphs/covid_graphs/country_graph.py:29]: That's formatting according to `yapf`

File: covid_graphs/covid_graphs/country_graph.py (hunk 2 above)
Comment (lukipuki) [reply covid_graphs/covid_graphs/country_graph.py]: 80k is confusing to general public.

I'd prefer something along
```
d3.format(",.2r")(4223);  // grouped thousands with two significant digits, "4,200"
```

It has a divider (a comma) and does some rounding. Though I'd skip the rounding.

File: covid_graphs/covid_graphs/scripts/country_dashboard.py (hunk 3 above)
Comment (lukipuki) [reply covid_graphs/covid_graphs/scripts/country_dashboard.py]: Will try it out tomorrow.

File: covid_graphs/covid_graphs/country_graph.py (hunk 4 above)
Comment (lukipuki) [reply covid_graphs/covid_graphs/country_graph.py:71]: Yes, but the `yaxis` range should be also reduced to `[1, max_value]`. Without it it looks even weirder.

**Pull request merged**